.github/workflows/build-and-test.yml

```diff
@@ -27,9 +27,22 @@ jobs:
           key: v1-${{ runner.os }}-gradle-${{ github.ref }}-${{ github.sha }}
           restore-keys: v1-${{ runner.os }}-gradle-${{ github.ref }}
 
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v2
```

Comment:
Is there a performance benefit for using this cache action? We don't use `actions/cache` in our workflows and I'm wondering if we should start doing it. The sonar scan in our build takes less than 1min so it's already pretty fast.

Reply:
Whatever works. This came in with the template action provided by SonarCloud but you're welcome to change it. I don't see why there would be a benefit to caching actually if it's running one scan on an action runner that's going to be thrown away.